### DISCUSSION TITLE
Is there a trick to getting notes to open in tab in Zotero 7?

### DISCUSSION THREAD

**Discussion by sicahjes** [windingwind/zotero-better-notes#1008] [Q&A]
For the life of me, I cannot get notes to open in a tab properly. The way it functions depends on whether I have ‘_Take over opening note_’ enabled in the settings, but regardless it doesn’t work:

**Enabled**

- Clicking on a note just opens the note in the side pane
- Double clicking opens as a seperate window (so annoying with stage manager on)
- If I select ‘_Open as tab_’ from the ellipsis menu in the side pane, I get a blank tab.

![Screenshot 2024-06-25 at 16 45 00](https://github.com/windingwind/zotero-better-notes/assets/28281659/4be6ece4-d0a2-4750-aa17-ce70556ce1bc)

**Disabled**
Weirdly, when '_Take over opening note’_ is disabled, the ‘_Open as tab’_ option opens the note in a new window. 

Any ideas? In principle, this seems like a great tool... but if I can’t have centralised notes that open in a tab, it’s useless to me.

**Comment by windingwind** [comment]
What's your Zotero and plugin version? Always run the latest version of both if you are on beta.

**Reply by sicahjes** [reply]
I was using the most recent versions available at the time of writing (for both Zotero and the plugin), but I can see a newer version of the plugin is available as of 2hrs ago. 

I've just updated from Release v1.1.4-beta.87 to Release v1.1.4-beta.88 and the plugin is now working (double click opens the note in a new tab). However, I'm on a different machine today (running Sonoma, the issue arose on Ventura). I'll check to see if updating the plugin works on the other computer and post an update. 

Thanks
